src/fmt_md.rs

```diff
@@ -1790,14 +1791,15 @@ pub mod tests {
 
         #[test]
         fn single_line() {
-            check_render(
-                vec![
-                    MdElem::Inline(Inline::Footnote(Footnote {
-                        label: "a".to_string(),
-                        text: md_elems!["Hello, world."],
-                    })),
-                    m_node!(MdElem::ThematicBreak),
-                ],
+            check_render_with_ctx(
+                &MdOptions::default_for_tests(),
+                (
+                    MdContext::empty().with("a", md_elems!["Hello, world."]),
+                    vec![
+                        MdElem::Inline(Inline::Footnote("a".into())),
```

Comment:
❔ `"a".into()` is a bit non-obvious — I'd prefer a bespoke method that makes it clearer what's going on. (c.f. `into()` or `to_owned()` vs `to_string()`, for str types)